Formula/android-sdk.rb

```diff
@@ -117,6 +125,7 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert_match version.to_s, shell_output("#{prefix}/tools/emulator -version")
+    system bin/"adb version"
```

Comment:
this is actually spawning another shell. should be
```
system bin/"adb", "version"
```
or
```
system "#{bin}/adb", "version"
```
Ditto for the one below.